### ISSUE TITLE
1.50 release tarball has broken symlinks

### ISSUE THREAD

**Issue by saltedcoffii** [espeak-ng/espeak-ng#911]
On a chromeos machine where the install prefix is `/usr/local` instead of `/usr`, all of the autotools scripts and files that are symlinked in instead of copied in are broken. the broken symlinks are as follows:
```
INSTALL -> /usr/share/automake-1.15/INSTALL
compile -> /usr/share/automake-1.15/compile
config.guess -> /usr/share/automake-1.15/config.guess
config.sub -> /usr/share/automake-1.15/config.sub
depcomp -> /usr/share/automake-1.15/depcomp
install-sh -> /usr/share/automake-1.15/install-sh
ltmain.sh -> /usr/share/libtool/build-aux/ltmain.sh
missing -> /usr/share/automake-1.15/missing
```
Also in this list is every file in `m4/`.

To fix: please copy (or hard link) in these files instead of symlinking them.

**Comment by ryandesign** [comment]
I believe this was already fixed in 0328b8ed065738bbd5880ededb56ce68b7e9ab90 but there has been no new release since then.

**Comment by ryandesign** [comment]
Also, this is covered by #683. So this issue should be closed.
